src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2082,6 +2082,50 @@ install_deployment_kernel (OstreeSysroot   *sysroot,
   g_autofree char *options_key = ostree_kernel_args_to_string (kargs);
   ostree_bootconfig_parser_set (bootconfig, "options", options_key);
 
+  g_autoptr(GError) local_error = NULL;
+  GKeyFile *config = ostree_repo_get_config (repo);
+  gchar **read_values = g_key_file_get_string_list (config, "sysroot", "bls-append-except-default", NULL, &local_error);
```

Comment:
Also while we're here...I think we should move all parsing this value into `reload_sysroot_config()` so that any failures in parsing the value are surfaced when the repo config is loaded, and not when we're going to install the kernel.

(This would have made the CI failure a lot more obvious, and also I think the unit tests would fail in the same way)

Reply:
Addressed in https://github.com/ostreedev/ostree/pull/2666